renovate.json

```diff
@@ -68,8 +68,21 @@
       "groupName": "support-deps",
       "commitMessageTopic": "support dependencies"
     },
+    {
+      "matchFileNames": ["package.json", "package-lock.json"],
+      "groupName": "support-deps",
+      "matchDepTypes": ["devDependencies"],
+      "commitMessageTopic": "support dependencies"
+    },
+    {
+      "matchFileNames": ["package.json", "package-lock.json"],
+      "groupName": "operator-deps",
+      "excludeDepNames": ["pepr"],
+      "commitMessageTopic": "operator-deps"
+    },
```

Comment:
Did you happen to test this part locally/on a fork? I think it may need to be reordered - with this config the last match will win out. So any dev dependencies would actually be globbed up by the `operator-deps` match if I'm not mistaken?